packages/react-ui/internal/CustomComboBox/ComboBoxMenu.tsx

```diff
@@ -100,17 +101,21 @@ export class ComboBoxMenu<T> extends Component<ComboBoxMenuProps<T>> {
     }
 
     let total = null;
-    if (items && renderTotalCount && totalCount && items.length < totalCount) {
+    const renderedItems = items && items.map(this.renderItem);
+
+    if (renderedItems && renderTotalCount && totalCount && renderedItems.length < totalCount) {
```

Comment:
По идее, сообщение должно выводится, когда реальных элементов меньше чем totalCount. 

```suggestion
    if (renderedItems && renderTotalCount && totalCount && countItems.length < totalCount) {
```

В остальном - все круто.